README.md

```diff
@@ -24,14 +15,118 @@ Its meetings and membership are largely focused on the direction, design, and wo
+  * Description: Packages for underwater network wireless communication.
+  * Motive: Since underwater communications relies on acoustic waves, there are some specificities regarding these types of waves that should be taken into account during data transmission.
   * Repositories
-    * link-to-repository
-
-}}
+    * Suggestions...
+  * References: 
+    *  [ns uan](https://www.nsnam.org/docs/models/html/uan.html)
+
+* Common Messages
+  * Description: Specifics ROS messages for maritime robotics.
+  * Motive: There are maritime robotics sensors that gives specific type of messages.
+  * Unique Repo
+    *  Assist the development of [UW APL Hydrographic Messages](https://github.com/apl-ocean-engineering/hydrographic_msgs)
+
+* Drivers
+  * Description: Documentation and index for underwater sensors and actuators drivers.
+  * Motive: There are specifics components used only for maritime robotics, which requires drivers to interact with them. Although, there already is an ROS organization to develop ROS drivers, so instead of developing inside this organization, we would use it just to have an index describing maritimate focused ROS drivers.
+  * Unique Repo
+    * **[Proposal]** ROS Maritime Robotics Drivers Index
+    * **[Proposal]** Sonar and radar drivers
+  * References: 
+    *  [ROS Drivers](https://github.com/ros-drivers)
+
+* Documentation
+  * Description: Documentation and index for maritime robotics packages and organizations.
+  * Motive: There are already many organizations and repos for maritime robotics. It is important to map what it is already developed in the area as well as make tutorials for newcomers or people that want to contribute with maritime robotics development.
+  * Repositories
+    * **[Proposal]** ROS Maritime Robotics Documentation Index
+    * **[Proposal]** ROS Maritime Robotics Tutorials
+  * References: 
+    * [COLA2](https://iquarobotics.com/cola2): ROS and Gazebo
+    * [VRX](https://github.com/osrf/vrx): ROS and Gazebo
+    * [UUV Simulator](https://uuvsimulator.github.io/): ROS and Gazebo
+    * [Plankton](https://github.com/Liquid-ai/Plankton): ROS2 and Gazebo
+    * [Ignition native underwater vehicles](https://ignitionrobotics.org/api/gazebo/5.0/underwater_vehicles.html): Ignition 
+    * [MBARI Vehicle Simulated in Ignition](https://github.com/osrf/lrauv/): Ignition 
+    * [Project Dave](https://github.com/Field-Robotics-Lab/dave/wiki): ROS and Gazebo
+    * [DS SIM](https://bitbucket.org/whoidsl/ds_sim/src/master/): ROS and Gazebo
+    * [UW APL Hydrographic Messages](https://github.com/apl-ocean-engineering/hydrographic_msgs): ROS
+  
+* Dynamic Model
+  * Description: Packages related to estimation and iteration of maritime robots models.
+  * Motive: Maritime robots dynamical models have to take into account buoayancy, hydrodynamics... Therefore some packages regarding math models are needed. 
+  * Repositories
+    * **[Proposal]** Maritime vehicle model estimation based on URDF
```

Comment:
No... Maybe I wasn't very clear. The model estimation with URDF it would be using tools for calculating the parameters using the meshes. The other would be based on experiments. The last one is just a model of the vehicle node to receive the current states, a period of time and then predict the next states based on the equations.